<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Material Definition</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="toolHeadLabel">
               <property name="text">
                <string>Tool Head</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="toolHeadComboBox">
               <item>
                <property name="text">
                 <string>Valve</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Displacement</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Plastic</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="tipLabel_2">
               <property name="text">
                <string>Material Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="matNameLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>default</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="tipLabel_3">
               <property name="text">
                <string>Variable name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="matVarLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>default</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Material Calibration</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="widthLabel">
               <property name="text">
                <string>Path Width</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="widthSpinBox">
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="minimum">
                <double>0.010000000000000</double>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>0.800000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="heightLabel">
               <property name="text">
                <string>Path Height</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="heightSpinBox">
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>0.800000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="speedLabel">
               <property name="text">
                <string>Path Speed</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="speedSpinBox">
               <property name="suffix">
                <string> mm/s</string>
               </property>
               <property name="minimum">
                <double>0.010000000000000</double>
               </property>
               <property name="maximum">
                <double>120.000000000000000</double>
               </property>
               <property name="value">
                <double>30.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="areaLabel">
               <property name="text">
                <string>Area Constant</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="areaSpinBox">
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="maximum">
                <double>1.500000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="cvLabel">
               <property name="text">
                <string>Compression Volume</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="cvSpinBox">
               <property name="suffix">
                <string> mm^3</string>
               </property>
               <property name="decimals">
                <number>1</number>
               </property>
               <property name="maximum">
                <double>1000.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>10.000000000000000</double>
               </property>
               <property name="value">
                <double>0.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="tipLabel">
               <property name="text">
                <string>Tip</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="tipLineEdit">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>blue Taper</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="title">
           <string>Tool head dependant info</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="psiLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>PSI</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="psiSpinBox">
               <property name="prefix">
                <string/>
               </property>
               <property name="suffix">
                <string> PSI</string>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>5.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="tempLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Temprature</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="tempSpinBox">
               <property name="suffix">
                <string> C</string>
               </property>
               <property name="decimals">
                <number>0</number>
               </property>
               <property name="minimum">
                <double>180.000000000000000</double>
               </property>
               <property name="maximum">
                <double>310.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>10.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Comment</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="commentTextEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QPushButton" name="xdflButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Generate Test XDFL</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="toolScriptButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string>Generate Toolscript</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
